### ISSUE TITLE
Support for php

### ISSUE THREAD

**Issue by Gspohu** [CoatiSoftware/CoatiBugTracker#200]
Hi,

It would be nice that coati support php.
Please add it to the roadmap.

Thank for readding,

**Comment by egraether** [comment]
+1 by mail

**Comment by egraether** [comment]
+1 by feedback

**Comment by bakneo** [comment]
yes Php as well as Salesforce, php it is also becoming a standard in web developing

**Comment by egraether** [comment]
Since we cannot keep up with language requests, we created the [SourcetrailDB](https://github.com/CoatiSoftware/SourcetrailDB) library to allow anyone to write custom indexers and extend language support for Sourcetrail.

If you are interested in working on an indexer for PHP, please let us know!

**Comment by XVilka** [comment]
#750 might be the way to go

**Comment by itsnotvalid** [comment]
@XVilka apparently no import graph and call graph for PHP from https://github.com/github/semantic

**Comment by Mte90** [comment]
Any updates?
